tests/meson.build

```diff
@@ -1,13 +1,44 @@
+
 if get_option('tests')
+    configure_file (
+        input: 'config.vapi.in',
+        output: 'config.vapi',
+        copy: true
+    )
+
+    conf = configuration_data()
+    conf.set_quoted ('TEST_PACKAGE_FILE', meson.current_source_dir() / 'assets' / 'vamp.json')
+    conf.set_quoted ('FULL_TEST_PACKAGE_FILE', meson.current_source_dir() / 'assets' / 'full-vamp.json')
+    conf.set_quoted ('BASIC_EXPECTED_TEST_PACKAGE_FILE', meson.current_source_dir() / 'assets' / 'expected-basic-vamp.json')
+    conf.set_quoted ('FULL_EXPECTED_TEST_PACKAGE_FILE', meson.current_source_dir() / 'assets' / 'expected-full-vamp.json')
```

Comment:
Two things:
1. You do not need to use the `/` syntax to concatenate strings
2. Meson already has support for literal maps in `configure_file()`, do we still need to create the configuration manually?